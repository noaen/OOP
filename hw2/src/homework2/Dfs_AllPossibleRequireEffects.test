# Sample Testing file

# Create a graph
CreateGraph graph1

# Create nodes
CreateNode n1 5
CreateNode n2 5
# Add nodes
AddNode graph1 n1
AddNode graph1 n2

# Add an edge
AddEdge graph1 n1 n2

# Dfs Algorithm
DfsAlgorithm graph1 n1
DfsAlgorithm graph1 n1 n2
DfsAlgorithm graph1 n2
DfsAlgorithm graph1 n2 n1

# no graph with name
DfsAlgorithm graph2 n1

# no node with name
DfsAlgorithm graph1 n3

# node does not exist in graph
CreateNode n3 5
DfsAlgorithm graph1 n3
