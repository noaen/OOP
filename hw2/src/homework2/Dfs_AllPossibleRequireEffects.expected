# Sample Testing file

# Create a graph
created graph graph1

# Create nodes
created node n1 with cost 5
created node n2 with cost 5
# Add nodes
added node n1 to graph1
added node n2 to graph1

# Add an edge
added edge from n1 to n2 in graph1

# Dfs Algorithm
dfs algorithm output graph1 n1: n1 n2
dfs algorithm output graph1 n1 -> n2: n1 n2
dfs algorithm output graph1 n2: n2
dfs algorithm output graph1 n2 -> n1: no path was found

# no graph with name
no graph with name graph2 exists

# no node with name
no node with name n3 exists

# node does not exist in graph
created node n3 with cost 5
node n3 does not exist in graph graph1
